cdap-docs/developers-manual/source/advanced/etl.rst

```diff
@@ -0,0 +1,85 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+.. _advanced-custom-etl:
+
+===============================================
+Creating Custom ETL Applications and Components
+===============================================
+
+Overview
+========
+- Purpose: For people writing new ETL Applications and ETL Components
```

Comment:
We are not going to use ETL Applications, ETL Components.

ETL Adapters. ETL Plugins.